newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -169,6 +171,10 @@ export const LeaderboardAdmin = ({ onLoading, project }: Props) => {
           ? 'leaderboardNameUpdate'
           : payload.sort
           ? 'leaderboardSortUpdate'
+          : payload.visibility
```

Comment:
maybe this could become a small function `getApiError(payload)` with a switch case?
returning :
`{ action: xxx, message: yyy}`